aten/src/ATen/templates/TensorBody.h

```diff
@@ -339,16 +340,16 @@ class CAFFE2_API Tensor {
   }
 
   template <typename T>
-  T * data_ptr() const;
+  TORCH_API T * data_ptr() const;
```

Comment:
Hmm, that's weird. The `API` annotation on the class itself should have been sufficient. What's going on here?

Reply:
err, will update; I was debugging a windows build error